meteor/sofie-server/business-logic/services/interfaces/rundown-event-builder.ts

```diff
@@ -3,7 +3,7 @@ import {
 	AdLibPieceInsertedEvent,
 	InfiniteRundownPieceAddedEvent,
 	RundownEvent,
-} from '../../../model/interfaces/rundown-event'
+} from '../../../model/value-objects/rundown-event'
```

Comment:
I'm fine with the change, but i would like to know the reason for it?

Reply:
Because it might not only be interfaces. And like the entities folder, which is called what it contains, this folder should hold value objects, whether they be described as interfaces or other technical constructs.

Reply:
Ok. Would that mean that interfaces and enums would be placed in the same folder?